hack/multiplatform-build.sh

```diff
@@ -45,9 +45,9 @@ for PLATFORM in ${PLATFORMS} ; do
   FULL_NAME=${BUILD_NAME}-${GOOS}-${GOARCH}
   ${GO_BUILD_ENV} ${GO_CMD} build -ldflags="${LD_FLAGS}" -o ${BUILD_PATH}/${FULL_NAME}${EXTENSION} $1
 
-  mkdir -p ${BUILD_PATH}/tmp/${FULL_NAME}
-  cp ${ROOT_PATH}/LICENSE ${BUILD_PATH}/tmp/${FULL_NAME}
-  cp ${BUILD_PATH}/${FULL_NAME}${EXTENSION} ${BUILD_PATH}/tmp/${FULL_NAME}/${BUILD_NAME}${EXTENSION}
-  (cd ${BUILD_PATH}/tmp && tar -czf ${BUILD_PATH}/${FULL_NAME}.tar.gz ${FULL_NAME})
+  mkdir ${BUILD_PATH}/tmp
```

Comment:
```suggestion
  mkdir -p ${BUILD_PATH}/tmp
```

Let's keep using the `-p` option.

Reply:
Done